### DISCUSSION TITLE
Improve behavior of displaying dropdown fields without a default value

### DISCUSSION THREAD

**Discussion by vaelu** [craftcms/cms#12235] [Ideas] [answered]
This discussion originates from #12227.

When you create a dropdown field and don't set a default value, the first value is "selected" by default when you create a new entry. But, technically, it's not (it has no `selected` attribute). This leads to behavior that, for example a condition which is based on `[belongs to value 1]`, will not work, only when you select another value and then select the first value again.

I think it would improve UX, if there would be a placeholder option like "Select an option", which is disabled by default. Also, it would throw an error if this option is selected and the field is required, because it has no real value.

Something like this (of course in a better design):
![image](https://user-images.githubusercontent.com/72358141/199125838-6dca744f-5ec6-4948-894a-d845ab7142d6.png)

**Comment by brandonkelly** [comment, marked as answer]
Resolved this for Craft 4.4 (e2eb6518133fd6e4ee1ba8f44dc4dbfd24d82847).

Going forward, if your Dropdown field doesn’t already have a blank option defined and is missing a value, a blank option will be prepended to the list and preselected (similar to how we resolved #9989).

The option won’t have a label. If you want to give it a label, you would need to define the option yourself in the Dropdown field’s settings. As long as the option doesn’t have a value, it won’t interfere with required field validation.

**Reply by brandonkelly** [reply]
4.4 is out!
